policy/modules/system/systemd.te

```diff
@@ -1194,6 +1194,7 @@ files_delete_usr_files(systemd_tmpfiles_t)
 files_list_home(systemd_tmpfiles_t)
 files_list_locks(systemd_tmpfiles_t)
 files_manage_generic_tmp_dirs(systemd_tmpfiles_t)
+files_manage_non_auth_files(systemd_tmpfiles_t)
```

Comment:
set the state of the `systemd_tmpfiles_manage_all` boolean to true instead

Reply:
Will do!